tests/test_viewers.py

```diff
@@ -107,6 +109,10 @@ def test_structure_data_viewer_storage(structure_data_object):
     # fmt: on
     v._render_structure()
 
+    # Make sure we don't polute current working dir with tempfiles
```

Comment:
Thanks for the fix, but is this necessary for the test to pass? If it pollutes the working dir, will the test fail?